<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestTriangleMainDialog</class>
 <widget class="QDialog" name="QtTestTriangleMainDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>812</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TestTriangle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolTestTriangle.qrc">
    <normaloff>:/ToolTestTriangle/ToolTestTriangleIcon.png</normaloff>:/ToolTestTriangle/ToolTestTriangleIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}

QWidget { font: bold, &quot;Courier New&quot;, courier, monospace }
QPushButton { font: 18 px  }
QTabWidget { font: 12 px  }
QLabel { font: 12px }
QComboBox { font: 12px }
QPlainTextEdit { font: 10px }

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="edit_shapes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string>#POLYGON((-1 -1 ,  1 -1 ,  1  1 , -1  1))
#POLYGON((-2 -2 ,  2 -2 ,  2  2 , -2  2))
#POLYGON((-3 -3 ,  3 -3 ,  3  3 , -3  3))
#POLYGON((-4 -4 ,  4 -4 ,  4  4 , -4  4))
POLYGON((-5 -5,-4 -5,-3 -5,-2 -5,-1 -5,0 -5,1 -5,2 -5,3 -5,4 -5,5 -5,5 -4,5 -3,5 -2,5 -1,5 0,5  1,5 2,5 3,5 4,5 5,4 5,3 5,2 5,1 5,0 5, -1 5,-2 5,-3 5,-4 5,-5 5,-5 4,-5 3,-5 2,-5 1,-5 0,-5 -1,-5 -2,-5 -3,-5 -4))
POLYGON((-10 -10,-8 -10,-6 -10,-4 -10,-2 -10,0 -10,2 -10,4 -10,6 -10,8 -10,10 -10,10 -8,10 -6,10 -4,10 -2,10 0,10 2,10 4,10 6,10 8,10 10,8 10,6 10,4 10,2 10,0 10,-2 10,-4 10,-6 10,-8 10, -10 10,-10 8,-10 6,-10 4,-10 2,-10 0,-10 -2,-10 -4,-10 -6,-10 -8))</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_shapes">
     <property name="text">
      <string>Shapes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_triangle">
     <property name="text">
      <string>Triangle</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_triangle_cpp">
     <property name="text">
      <string>TriangleC++</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="ribi::QtNavigationableGraphicsView" name="view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ribi::QtNavigationableGraphicsView" name="view_triangle_mesh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="ribi::QtNavigationableGraphicsView" name="view_triangle_cpp_mesh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_triangle_exe">
     <property name="text">
      <string>Triangle.exe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_triangle_cpp_exe">
     <property name="text">
      <string>TriangleCpp.exe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ribi::QtNavigationableGraphicsView" name="view_triangle_mesh_exe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="ribi::QtNavigationableGraphicsView" name="view_triangle_cpp_mesh_exe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="box_parameters">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_triangle_quality">
        <property name="text">
         <string>quality, maximum angle (degrees) </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="box_triangle_area">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>60.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_triangle_area">
        <property name="text">
         <string>area maximum (m2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="box_triangle_quality">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_verbose">
        <property name="text">
         <string>Verbose</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ribi::QtNavigationableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>qtnavigationablegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ToolTestTriangle.qrc"/>
 </resources>
 <connections/>
</ui>
